### ISSUE TITLE
Hardware is unsupported by NNPACK so falling back to default NNlib

### ISSUE THREAD

**Issue by denizyuret** [FluxML/NNlib.jl#225]
I get the following warning when I try `using NNlib`. How can I fix it?
```
┌ Warning: Hardware is unsupported by NNPACK so falling back to default NNlib
└ @ NNlib /userfiles/dyuret/.julia/packages/NNlib/sSn9M/src/nnpack/NNPACK.jl:42
```

**Comment by denizyuret** [comment]
Comment from @DhairyaLGandhi: Nnpack was something that was added as a performance optimisation for CPU targets. I don't think it's critical right now, but it adds more avx instruction usage among other things
The hardware support is listed https://github.com/Maratyszcza/NNPACK#platforms-and-requirements

**Comment by denizyuret** [comment]
I recommend this warning to be removed.

**Comment by CarloLucibello** [comment]
fixed in #227
